spec/controllers/documents_controller_spec.rb

```diff
@@ -208,6 +208,40 @@
         it_behaves_like "update verification type", "citizenship_type", reason, "verify", "lawful_presence_update_reason", reason
       end
     end
+
+    context 'admin_rejected a verification_type' do
+      let!(:person10) { FactoryBot.create(:person, :with_family, :with_consumer_role) }
+      let!(:user10) { FactoryBot.create(:user, person: person10) }
+      let!(:ver_type10) do
+        person10.verification_types.create!(type_name: 'Citizenship', validation_status: 'unverified')
+        person10.verification_types.where(type_name: 'Citizenship').last
+      end
+
+      let(:input_params) do
+        { person_id: person10.id,
+          verification_type: ver_type10.id,
+          admin_action: 'return_for_deficiency',
+          family_member_id: person10.primary_family.primary_applicant.id,
+          verification_reason: 'Illegible' }
+      end
+
+      before do
+        post :update_verification_type, params: input_params
+        ver_type10.reload
+      end
+
```

Comment:
can you merge below 3 expectations in to 1 to avoid 3 calls here. you can use `pluck`